src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -281,6 +281,15 @@ public void Send_ServerDoesntExist_Throws()
         }
 
         [Fact]
+        public async Task SendAsync_ServerDoesntExist_Throws()
+        {
+            using (var smtp = new SmtpClient(Guid.NewGuid().ToString("N")))
+            {
+                await Assert.ThrowsAsync<SmtpException>(() => smtp.SendMailAsync("anyone@anyone.com", "anyone@anyone.com", "subject", "body"));
```

Comment:
```
> telnet.exe anyone.com 25

220-mail.dreamissary.com ESMTP
220-MagicMail Daemon with Built-In Anti-Spam
220 See http://www.linuxmagic.com for info
QUIT
```

Looks like SMTP server is real, but test will pass  :smile:

Reply:
anyone.com is in the email address... it's not the server bring connected to. The server bring connected to is the result of Guid.NewGuid().ToString("N"), which does not exist.